src/SonarQube.Client/Messages/QualityProfileRequest.cs

```diff
@@ -24,8 +24,11 @@ namespace SonarQube.Client.Messages
 {
     public class QualityProfileRequest
     {
+        // TODO: since 6.5 this parameter is named "project"; we should change a couple of months after release of LTS.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)